jormungandr/src/network/subscription.rs

```diff
@@ -360,7 +350,7 @@ impl FragmentProcessor {
             debug!(logger, "error sending fragments for processing"; "reason" => %e);
             Error::new(Code::Internal, e)
         })?;
-        let fragments = self.buffered_fragments.split_off(0);
+        let fragments = mem::replace(&mut self.buffered_fragments, Vec::new());
```

Comment:
I would put an initial capacity of `buffer_sizes::inbound::FRAGMENTS`.

Reply:
Will do, but as the chunking is lazy the yielded vectors may be underfilled.